universal-login-web3/package.json

```diff
@@ -24,7 +24,7 @@
     "build": "tsc --build tsconfig.json",
     "lint": "tslint -p tsconfig.json \"{lib,test}/**/*.{ts,tsx}\"",
     "test": "yarn build && yarn test:nobuild",
-    "test:nobuild": "cross-env NODE_ENV=test mocha --require ./test/helpers/nullCompiler.ts  -t 10000 \"dist/test/**/*.js\"",
+    "test:nobuild": "cross-env NODE_ENV=test mocha --require ./test/helpers/nullCompiler.ts --exit -t 10000 \"dist/test/**/*.js\"",
```

Comment:
`--exit` flag stinks, do we have a test that never ends?

Reply:
From what i found out, adding `sdk.start()` creates a leak, even with `sdk.stop()` in after each

Reply:
we can use finalizeAndStop() instead